.prettierrc

```diff
@@ -0,0 +1,8 @@
+printWidth: 120
+tabWidth: 2 # Default
```

Comment:
@quesabe If we use the default values, it is worth adding them all, otherwise there is no point in adding them at all. I think it is better to add them all, since the default values change over time (e.g. https://prettier.io/docs/en/options.html#arrow-function-parentheses).

Reply:
I've added all the options, even though `vueIndentScriptAndStyle` might not be relevant.